src/grpc_generated/go/gen_go_stubs.sh

```diff
@@ -25,14 +25,20 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# Local
-PACKAGE_PATH="nvidia_inferenceserver"
-# Or add to GOPATH and remove "./" from import path in grpc_simple_client.go
-#PACKAGE_PATH="${GOPATH}/src/nvidia_inferenceserver"
+PACKAGE="grpc-client"
 
-mkdir -p ${PACKAGE_PATH}
+# Clean up before
+rm -rf ./core
 
 # Get the proto files from the common repo
 mkdir core && cp common/protobuf/*.proto core/.
+
+for i in ./core/*.proto
+do 
+  # https://developers.google.com/protocol-buffers/docs/reference/go-generated#package
+  echo "option go_package = \"./${PACKAGE}\";" >> $i
+done
+
 # Requires protoc and protoc-gen-go plugin: https://github.com/golang/protobuf#installation
-protoc -I core --go_out=plugins=grpc:${PACKAGE_PATH} core/*.proto
+# https://github.com/golang/protobuf/issues/1070.
```

Comment:
Linking to a GitHub issue to provide installation instructions does not seem like good practice. At a minimum, we need clearer context. What would be better is to link to installation instructions or provide them (@rmccorm4 can clarify whether the latter is bad practice, since they may go out of date). For example, perhaps I did it wrong, but I used `get go` and `get install` to get protoc-gen-go-grpc and protoc-gen-go, but then updated the script to use the absolute path (in hindsight, I could have added it to the PATH system variable). I'm new to Go, so perhaps this is clearer to Go users, but providing a link or clear steps to installing the necessary components would save user time and frustration, plus remove room for error.

Reply:
> Linking to a GitHub issue to provide installation instructions does not seem like good practice. At a minimum, we need to provide context. What would be better is to link to installation instructions or provide them (@rmccorm4 can clarify whether the latter is bad practice, since they may go out of date).

Good point, I should just have had it in the PR to give background for the change. Will drop the link. 

> 
>  I'm new to Go, so perhaps this is clearer to Go users, but providing a link or clear steps to installing the necessary components would save user time and frustration, plus remove room for error.

Hmm it would be nice, but is this consistent with how it is done for other parts? I kind of feel that it is outside the scope. However I will leave this up to you guys to decide. If you want install instructions I would be happy to supplied them.

Reply:
Will defer to @rmccorm4. It sounds like this change makes it so that this is different from standard Go installation, which is why I added the comment. If you need to install additional plugins to make the script work for Go, then it seems consistent to me.